README.md

```diff
@@ -110,17 +121,15 @@ This plugin defines hooks for `environment`, `checkout`, `command`, and `post-co
 
 #### Available properties
 
-- `input_artifacts`: String array of artifact paths to download and sign.
+- `input_artifact`: An artifact path to download and sign.
+- `sign_prerequisites`: (optional) String array of the specific artifact paths to sign. If empty, default to `input_artifact`. In the case of `.app`s, make sure to list these bottom-up; internal frameworks/helpers first, and `.app` at the end.
+- `entitlements`: String path of artifact containing a valid entitlements plist to apply.
```

Comment:
Is this optional? It doesn't seem to be specified in the examples, so I suspect it is.

If so, label it as such (as well as any other properties that are optional).